app/javascript/readingList/components/TagList.jsx

```diff
@@ -0,0 +1,88 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+function LargeScreenTagList({ availableTags, selectedTag, onSelectTag }) {
+  return (
+    <fieldset className="hidden grid grid-cols-1 gap-2">
+      <legend className="hidden">Filter by tag</legend>
+      {availableTags.map((tag) => (
+        <label
+          className={`crayons-link crayons-link--block${
+            selectedTag === tag ? ' crayons-link--current' : ''
+          }`}
+          aria-label={`${tag} tag`}
+        >
+          <input
+            type="radio"
+            name="filterTag"
+            onClick={onSelectTag}
+            key={tag}
+            className="opacity-0"
+            checked={selectedTag === tag}
+            value={tag}
+          />
+          #{tag}
+        </label>
+      ))}
+    </fieldset>
```

Comment:
Using radio input here is a nice hack but it makes the nav accessibility a bit weird: when using my keyboard [tab] it focus the first element only and then jumps straight to articles list.. when going back [shift+tab] it highlights the last element only... Since this is radio, I'd have to use arrows to navigate through these radio options but that seems a bit confusing.

https://user-images.githubusercontent.com/108287/110937032-15117080-8332-11eb-9863-2208ecf2f68f.mov

(I added red outline to highlight focused elements in my vide but for some reason it didn't apply to tags :P but i hope you can spot what I'm talking about)

My suggestion would be replacing radio elements with classic links (or at least buttons?) but I can imagine it will require a bit more refactoring. But having form elements here instead of regular links doesn't seem to be the right choice.

Reply:
Oh, I've just noticed your older comment about refactoring that in a separate PR. Sure. 

> In regards to keeping the filter in the URL, I completely agree, but that should be a separate PR a that changes how the entire state is managed for this component.

^ I agree but in this case I wouldn't refactor the nav to be a form radio inputs (they currently are links in prod), since we will circle back to this anyway and turn them to be links again - instead probably only make adjustments to the `onSelectTag` function so we can avoid that UX regression with keyboard navigation introduced by turning links into form elements.

Reply:
I agree with @ludwiczakpawel I think for a sighted user it's not clear how to navigate the tags when they don't look like radios. 

Thinking about it, at the moment it's almost behaving more like a set of Tabs, with a single active page/tab at a time, but that also doesn't feel intuitive with the UI. It's been suggested already, but making these links and having the filter in the URL makes a lot of sense, and when we do that work this list can become a secondary `nav`.

tl;dr I second @ludwiczakpawel's suggestion to keep them as they are until we make that change.

Reply:
I can put it back and disable the eslint rule that complains we're using a link like a button. Like already mentioned, the links in prod have click event listeners on them already.